src/internal_modules/roc_fec/reader.cpp

```diff
@@ -301,7 +312,8 @@ void Reader::fill_source_block_() {
             break;
         }
 
-        (void)source_queue_.read();
+        packet::PacketPtr ptr;
+        (void)source_queue_.read(ptr);
```

Comment:
In cases like this when no error can happen, we should add a panic instead of ignoring error.